modules/dashboard/tests/widgets-balancer/1-example.tf

```diff
@@ -0,0 +1,47 @@
+# data "aws_vpcs" "vpcs" {}
+
+# data "aws_subnet_ids" "subnets" {
+#   vpc_id = data.aws_vpcs.vpcs.ids[0]
+# }
+
+# module "alb" {
+#   source  = "terraform-aws-modules/alb/aws"
+#   version = "~> 6.0"
+
+#   name = "my-alb"
+
+#   load_balancer_type = "application"
+
+#   vpc_id  = data.aws_vpcs.vpcs.ids[0]
+#   subnets = data.aws_subnet_ids.subnets.ids
+# }
+
+module "dashboard-with-balancer-metrics" {
+  source = "../../"
+  name   = "dashboard-with-balancer-metrics"
+
+  defaults = {
+    period : 300
+  }
+
+  rows = [
+    # [
```

Comment:
commented code

Reply:
This one is on purpose, cannot yet figure out easy way to create dependant balancer.